presto-sqlserver/src/test/java/io/prestosql/plugin/sqlserver/SqlServerQueryRunner.java

```diff
@@ -97,4 +99,24 @@ private static Session createSession()
                 .setIdentity(new Identity(TestingSqlServer.USER, Optional.empty()))
                 .build();
     }
+
+    public static void main(String[] args)
+            throws Exception
+    {
+        Logging.initialize();
+
+        TestingSqlServer testingSqlServer = new TestingSqlServer();
+
+        // SqlServer is using docker container so in case that shutdown hook is not called, developer can easily clean docker container on their own
+        Runtime.getRuntime().addShutdownHook(new Thread(() -> testingSqlServer.close()));
+
+        DistributedQueryRunner queryRunner = (DistributedQueryRunner) createSqlServerQueryRunner(
+                testingSqlServer,
+                ImmutableMap.of(),
+                ImmutableList.of());
+        Thread.sleep(10);
```

Comment:
Can this be removed?

Reply:
I added sysout before and after:
```
2019-09-06T10:07:19.171+0200	INFO	main	io.prestosql.tests.DistributedQueryRunner	Created catalog sqlserver (sqlserver) in 148.13ms
2019-09-06T10:07:19.190+0200	INFO	main	io.prestosql.tests.DistributedQueryRunner	Announced catalog sqlserver (sqlserver) in 18.81ms
2019-09-06T10:07:19.451+0200	INFO	main	stdout	sleep start
2019-09-06T10:07:19.462+0200	INFO	main	stdout	sleep end
2019-09-06T10:07:19.462+0200	INFO	main	io.prestosql.tests.DistributedQueryRunner	======== SERVER STARTED ========
2019-09-06T10:07:19.462+0200	INFO	main	io.prestosql.tests.DistributedQueryRunner	
====
http://127.0.0.1:59168
====
```

looks like we may be doing nothing really during this time.